README.md

```diff
@@ -1,34 +1,129 @@
 ![OpenUI5 logo](http://openui5.org/images/OpenUI5_new_big_side.png)
 
 # openui5-sample-app
-> [OpenUI5](https://github.com/SAP/openui5) sample app using Grunt, Karma & Bower
+> [OpenUI5](https://github.com/SAP/openui5) sample app using the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling).
+
+## Prerequisites
+- The **UI5 CLI** of the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling#installing-the-ui5-cli).
+    - For installation instructions please see: [Installing the UI5 CLI](https://github.com/SAP/ui5-tooling#installing-the-ui5-cli).
 
 ## Getting started
-* Install Node.js (from [nodejs.org](http://nodejs.org/)).
-* Install the Grunt CLI
-    ```sh
-    npm install --global grunt-cli
-    ```
-* Clone the repository and navigate into it
+1. Clone the repository (on the `ui5-tooling` branch) and navigate into it
     ```sh
-    git clone https://github.com/SAP/openui5-sample-app.git
+    git clone -b ui5-tooling git@github.com:SAP/openui5-sample-app.git
     cd openui5-sample-app
     ```
-* Install all npm dependencies (also installs all bower dependencies)
+1. Install all dependencies
     ```sh
     npm install
     ```
 
-## Usage
-### Server
-Run `grunt serve` to start a local server with your application at [http://localhost:8080](http://localhost:8080).
+1. Start a local server and run the application (http://localhost:8080/index.html)
+    ```sh
+    ui5 serve -o /index.html
+    ```
+
+## Testing
+* Run ESLint code validation
+    ```sh
+    npm run lint
+    ```
+* Start a local server and execute the tests automatically after every change
+    ```sh
+    npm run watch
+    ```
+* Run ESLint, start a local server and run the tests in CI mode
+    ```sh
+    npm test
+    ```
+## Building
+### Option 1: Standard preload build
+1. Execute the build
+    ```sh
+    ui5 build -a
+    ```
+1. Run the result
+    1. Install an HTTP server like [zeit/serve](https://www.npmjs.com/package/serve) (**Note:** You can use any HTTP server)
```

Comment:
Due to a refactoring, in the latest versions of the `zeit/serve` module, no gzip is being applied anymore.

Details: https://zeit.co/blog/serve-7

Last known version with gzip support: `serve@6.5.3`

We should adapt our recommendation here to ensure a server with gzip support is being used.

Reply:
Reason for not having gzip in zeit/serve: https://github.com/zeit/serve-handler/issues/1#issuecomment-390632443